openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,32 +60,35 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A collection of one or more assertions about a subject, provided by a Credential Issuer. These assertions are also known as claims.
 
-Verifiable Credential (VC):
-:  An Issuer-signed Credential whose integrity can be cryptographically verified. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
+Verifiable Credential (or Credential):
+:  An Issuer-signed Credential Dataset whose integrity can be cryptographically verified. It can be of any Credential Format.
 
-W3C Verifiable Credential:
-:  A Verifiable Credential compliant to the [@VC_DATA] specification.
+Credential Format:
+:  A format used to represent a Verifiable Credential. The exact parameters required to describe a Credential Format are defined in the Credential Format Profile.
+
+Credential Format Profile:
+:  Sets of parameters specific to individual Credential Formats. Credential Format Profiles for W3C Verifiable Credentials [@VC_DATA] and ISO/IEC 18013-5 mDL [@ISO.18013-5] are provided in (#format_profiles). Other specifications or deployments can define their own Credential Format Profiles using the extension points defined in this specification.
```

Comment:
I feel like we could be more specific about the purpose of a profile here, e.g the intent of the profile is to be able to "issue" a credential of that format over this protocol right?

Reply:
I agree. I was completely misreading this definition in first place. 

The intention of a credential format profile is to customize VCI for use with credentials of that format. That should be stated along with a list of thing that needs to be defined, such as identifier, use in authz details, use in credential request and so on.